### ISSUE TITLE
Swagger browser certificate?

### ISSUE THREAD

**Issue by emanuelpalm** [eclipse-arrowhead/core-java-spring#112]
I was able to bring up a Docker cluster using the `/docker/docker-compose.yml` file. Now I want to access the Swagger interfaces of the systems via my browser. When I try, I get told that a browser certificate is required. Which certificate should I use? Could I simply add a new one to the trust stores of my systems, or is there something else I should do?

I want to run all services in secure mode. Switching to insecure mode is not an option.

**Comment by MaGaMeGa** [comment]
Please, let me know, if you tried sysop, as in : https://github.com/arrowhead-f/core-java-spring/issues/79.

**Comment by emanuelpalm** [comment]
I did previously try the certificate in `core-java-spring/certificates/testcloud1/sysop.p12`, which did not work. When changing to the certificate in `core-java-spring/serviceregistry/src/main/resources/sysop.p12`, as suggested in #79, the error message disappeared. However, I get this screen instead:

![Screenshot from 2019-10-15 10-24-05 cut](https://user-images.githubusercontent.com/1773338/66814084-4cf66780-ef36-11e9-9d7a-21dbaacb1e5c.png)

I've tried entering the following URL:s into the alert box without any other result than the box reappearing. 

- https://127.0.0.1:8443
- https://127.0.0.1:8443/api
- https://127.0.0.1:8443/swagger-ui.html
